ayon_bin_distro/lakectlpy/wrapper.py

```diff
@@ -42,25 +53,45 @@ def __init__(
             self.base_uri = None
 
         if access_key_id:
-            os.environ["LAKECTL_CREDENTIALS_ACCESS_KEY_ID"] = access_key_id
+            self.lake_ctl_acces_key_id = access_key_id
+            # os.environ["LAKECTL_CREDENTIALS_ACCESS_KEY_ID"] = access_key_id
         if secret_access_key:
-            os.environ["LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY"] = secret_access_key
+            self.lake_ctl_secret_acces_key = secret_access_key
+            # os.environ["LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY"] = secret_access_key
         if server_url:
-            os.environ["LAKECTL_SERVER_ENDPOINT_URL"] = server_url
+            self.lake_ctl_server_url = server_url
+            # os.environ["LAKECTL_SERVER_ENDPOINT_URL"] = server_url
 
         self.data = ""
 
     def _run(
         self, lakectl_command: list, cwd=None, non_blocking_stdout: bool = True
     ) -> subprocess.Popen:
+        """call lakectl bin with args gets lakectl bin, base uri and config from the member vars
+
+        Args:
+            cwd (): option to move the path from where we call the lakectl passed to subprocess.Popen
```

Comment:
Either add the type hint or remove the brackets.
```suggestion
            cwd: option to move the path from where we call the lakectl passed to subprocess.Popen
```